src/cljs/main/broadfcui/page/workspace/notebooks/create_cluster.cljs

```diff
@@ -41,6 +41,10 @@
                    "n1-highmem-64"
                    "n1-highmem-96"])
 
+; TODO: FireCloud temporarily pegged at an older Jupyter image with Hail 0.1
+; See https://github.com/DataBiosphere/leonardo/issues/699
+(def jupyterDockerImage "us.gcr.io/broad-dsp-gcr-public/leonardo-notebooks:hail0.1")
```

Comment:
The `hail0.1` tag exists here: https://console.cloud.google.com/gcr/images/broad-dsp-gcr-public/US/leonardo-notebooks